### PR TITLE
Updating atravitz profile

### PR DESCRIPTION
**Pull request by atravitz**
adding my bio. Also added "order" values for those of us recently added.

### CODE REVIEW

**Comment by IAlibay** [comment]
@atravitz what does the order value do? I.e. do we need to add one for other folks?

File: _team/riesben.md [hunk 1]
```diff
@@ -9,5 +9,5 @@ blurb: >-
   He studied Biochemistry and Bioinformatics at the Eberhard-Karls University Tübingen and worked on free energy calculation methods during his PhD at ETH Zurich.
   His work focused on facilitating the easy and efficient use of FE methods.
   During his leisure time, Benjamin hikes, bikes, or climbs mountains in his vicinity.
-order: 1
```
Comment (IAlibay) [inline _team/riesben.md:12]: Should we do the same for David Swenson?

**Review by IAlibay — APPROVED** [review]
Just one question, otherwise lgtm!

**Comment by atravitz** [comment]
> @atravitz what does the order value do? I.e. do we need to add one for other folks?

It determines order of our profiles, with alphabetical by file name "tie-breaking"

**Pull request merged**